### PR TITLE
Update the Readme & License

### PR DESCRIPTION
**Pull request by chrisfinazzo**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2014, caskroom
+Copyright (c) 2016, Caskroom
```
Comment (vitorgalvao) [inline LICENSE]: The copyright notice, although mostly meaningless, should be either the year of creation, or a range *from* that year (2014–2016). Please revert this change.

File: README.md [hunk 2]
```diff
@@ -1,5 +1,6 @@
 # http://caskroom.io
-The website of [homebrew-cask](https://github.com/phinze/homebrew-cask), a command-line interface for the administration of Mac applications.
+The website of [Homebrew Cask](https://github.com/phinze/homebrew-cask), a
```
Comment (vitorgalvao) [inline README.md]: Don’t break the line. Lines should be continuous.

File: README.md [hunk 3]
```diff
@@ -8,9 +9,13 @@ To run the site locally:
 ```shell
 $ git clone https://github.com/caskroom/caskroom.github.io.git
 $ cd caskroom.github.io
-$ jekyll server -w
+$ bundle exec jekyll build
+$ bundle exec jekyll serve
 ```
+To use this method, please install Jekyll. Bundler is optional, but recommended
```
Comment (vitorgalvao) [inline README.md]: Don’t break the line. Lines should be continuous.

File: README.md [hunk 4]
```diff
@@ -8,9 +9,13 @@ To run the site locally:
...
+for managing Jekyll and its related gem dependencies.
 
-The current codebase is not specific to jekyll. You can test it with your live-reload tool of choice, or simply host it with a Python server:
+The current codebase is not specific to jekyll. You can also test it with your
```
Comment (vitorgalvao) [inline README.md]: Don’t break the line. Lines should be continuous.

File: README.md (hunk 2 above)
Comment (chrisfinazzo) [inline README.md]: On other projects I've worked on, there has usually been a rule (either enforced through CI or at the maintainer's request) that lines not exceed a certain number of characters. For @jekyll/jekyll, the limit on documentation has been 80, for the gem itself, it is somewhere between 110 and 120 if memory serves. That was my intention with the changes in these files - do you have a preference in this regard?

Comment (vitorgalvao) [inline README.md]: The preference is that lines are not broken due to an arbitrary character limit. There’s no reason to, since the markdown files are displayed in places where reflow is done automatically.

File: LICENSE [hunk 5]
```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2014, caskroom
+Copyright (c) 2014 - 2016, Caskroom
```
Comment (vitorgalvao) [inline LICENSE]: Should be `2014–2016` (not only does spacing matters, but the actual separation character). We (well, mostly me, I guess) care about typographic correctness.

**Comment by vitorgalvao** [comment]
There’s still one fix pending. Also, there are too many commits. Commits reverting changes you made should not exist.

**Comment by vitorgalvao** [comment]
Do you intend to submit a new PR, or should I rebuild what you’ve done here and take the good parts.

**Comment by chrisfinazzo** [comment]
No, I'll clean things up and submit a separate PR. At that point, I'll close this and link to the new one.

**Comment by vitorgalvao** [comment]
Thank you.

**Comment by chrisfinazzo** [comment]
See #49.
